lib/src/lib.rs

```diff
@@ -635,23 +641,28 @@ impl Repo {
             path: repo_path.to_owned(),
             pub_key: pk,
             version: 0,
+            chunking: config::RepoChunking::default(),
         };
         Ok(repo)
     }
 
     /// Create new rdedup repository
-    pub fn init(repo_path: &Path, passphrase: &str) -> Result<Repo> {
+    pub fn init(repo_path: &Path,
+                passphrase: &str,
+                chunking: config::RepoChunking)
```

Comment:
Similarly here, I think we can already take whole `config::Repo`. Since it has a `Default` implemented, users of this API, will just create a default Config and modify the settings that they need to change. Or a builder pattern could be introduced.